selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -401,8 +404,11 @@ public void selectItem(String path, int timeoutForWaitingItem) {
       new WebDriverWait(seleniumWebDriver, EXPECTED_MESS_IN_CONSOLE_SEC)
           .until(ExpectedConditions.visibilityOfElementLocated(By.xpath(locator)))
           .click();
-    } catch (StaleElementReferenceException ex) {
-      LOG.error(ex.getLocalizedMessage(), ex);
+    }
+    // sometimes an element in the project explorer may be is not attached to the DOM. We should
```

Comment:
correct variant is `may not be attached`.
Could you, please, fix it in the whole class?